subprojects/core/src/integTest/groovy/org/gradle/internal/operations/logging/LoggingBuildOperationProgressIntegTest.groovy

```diff
@@ -427,14 +428,18 @@ class LoggingBuildOperationProgressIntegTest extends AbstractIntegrationSpec {
             .flatten()
             .with { it as List<BuildOperationRecord.Progress> }
             .findAll { OutputEvent.isAssignableFrom(it.detailsType) }
-            // Ignore deprecations, these are checked by the testing infrastructure.
+        // Ignore deprecations, these are checked by the testing infrastructure.
             .findAll { it.details.get("category") != LoggingDeprecatedFeatureHandler.class.name }
 
         // 11 tasks + "\n" + "BUILD SUCCESSFUL" + "2 actionable tasks: 2 executed"
         // when configuration cache is enabled also "Configuration cache entry reused." and "Parallel Configuration Cache is an incubating feature."
```

Comment:
💅  it would be clearer to have
`when configuration cache is enabled also "Configuration cache entry reused." and "Parallel Configuration Cache is an incubating feature."` in the new function rather than here.